addons/rose/addon/components/rose/form/radio/group/index.hbs

```diff
@@ -1,3 +1,6 @@
-{{yield (hash
+<div class="radio-field-wrapper {{if @inline "inline"}}">
```

Comment:
Can we get by without a wrapper?  The cards don't necessarily need to use flex and can always be inline.

Reply:
updated!